File: Api/ClientApp/src/app/models/salary.ratio.ts
export class SalaryRatio {
    experienceBonus: number;
    experienceBonusMaximum: number;
    subordinateBonus: number;
    
    constructor() {
    }
    
    static fromData(other: any): SalaryRatio {
        let ratio = new SalaryRatio();
        ratio.experienceBonus = Number(other.experienceBonus) || 0;
        ratio.experienceBonusMaximum = Number(other.experienceBonusMaximum) || 0;
        ratio.subordinateBonus = Number(other.subordinateBonus) || 0;
        return ratio;
    }
}

File: SalaryCounter.Service/Model/SalaryRatio.cs
namespace SalaryCounter.Service.Model
{
    // ReSharper disable once ClassWithVirtualMembersNeverInherited.Global
    public class SalaryRatio
    {
        public virtual decimal ExperienceBonus { get; set; }
        public virtual decimal ExperienceBonusMaximum { get; set; }
        public virtual decimal SubordinateBonus { get; set; }
    }
}